packages/modules/desktop-viewer-react/package.json

```
@@ -78,7 +78,7 @@
     "@itwin/core-frontend": "^4.0.0",
     "@itwin/ecschema-metadata": "^4.0.0",
     "@itwin/electron-authorization": "^0.13.0",
-    "electron": "^24.0.0",
+    "electron": "^24.8.3",
```

```suggestion
    "electron": "^24.0.0",
```

updating peers is breaking. we just need to update the dev dep in this pkg